### DISCUSSION TITLE
Question about consistency.

### DISCUSSION THREAD

**Discussion by Hyungsik-Jung** [shap/shap#1636] [Q&A]
Hi, I have questions about consistency property.
As an additive feature attribution methods, DeepLIFT satisfies local accuracy and missingness while defying consistency.

What does it mean that "not satisfying consistency" for attribution methods? Does only Shapley value satisfy the property?

Thank you for your help in advance.

**Comment by rproner1** [comment]
From the Lundberg and Lee (2017) paper, Shapley values are the only solution to the explanation model which satisfies all three properties. Consistency says that if the model changes such that the marginal contribution of feature j increases or stays the same over all possible subsets of features, then the Shapley value for feature j must not decrease. In other words, the direction of change in the Shapley value is _consistent_ with the direction of change in the marginal contribution. It would not make sense if a feature becomes more important after a function change but its Shapley value decreases.
